scio-avro/src/main/scala/com/spotify/scio/avro/types/MacroUtil.scala

```diff
@@ -47,9 +47,9 @@ private[types] object MacroUtil {
 
   // Namespace helpers
 
-  val ScioAvro = "_root_.com.spotify.scio.avro"
-  val ApacheAvro = "_root_.org.apache.avro"
-  val ScioAvroType = s"$ScioAvro.types.AvroType"
+  val ScioAvro: String = "_root_.com.spotify.scio.avro"
```

Comment:
Requiring explicit type ascription for basic `val` that are easily inferred seems excessive. IIRC this was not required when we had scalastyle doing the validation.